app/react-components/Mutation.js

```diff
@@ -1,15 +1,17 @@
+/* global $ */
```

Comment:
We do have it https://github.com/NCI-GDC/portal-ui/blob/next/package.json#L118
I'm for removing it entirely